pkgs/applications/networking/cluster/sonobuoy/default.nix

```diff
@@ -38,6 +38,6 @@ buildGoPackage rec {
 
     homepage = "https://github.com/heptio/sonobuoy";
```

Comment:
Could you also update this? I guess https://sonobuoy.io is the true homepage.

Reply:
Yes, fixed that.